### PR TITLE
New Quote added

### PR DESCRIPTION
**Pull request by SameerMakhija**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/quote-of-the-day/life/life2.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+“It doesn’t matter where you came from. All that matters is where you are going.”
```
Comment (casey-collab) [inline english/quote-of-the-day/life/life2.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**